pkg/cmd/workspace/stop.go

```diff
@@ -72,4 +80,27 @@ func init() {
 	}
 
 	StopCmd.Flags().StringVarP(&stopProjectFlag, "project", "p", "", "Stop a single project in the workspace (project name)")
+	StopCmd.Flags().BoolVarP(&allFlag, "all", "a", false, "Stop all workspaces")
+}
+
+func stopAllWorkspaces() error{
+	ctx := context.Background()
+	apiClient, err := server.GetApiClient(nil)
+	if err != nil {
+		return err
+	}
+
+	workspaceList, res, err := apiClient.WorkspaceAPI.ListWorkspaces(ctx).Execute()
+	if err != nil {
+		return apiclient.HandleErrorResponse(res, err)
+	}
+
+	for _, workspace := range workspaceList {
+		res, err := apiClient.WorkspaceAPI.StopWorkspace(ctx, *workspace.Id).Execute()
+		if err != nil {
+			return fmt.Errorf("failed to stop workspace %s: %v", *workspace.Id, apiclient.HandleErrorResponse(res, err))
```

Comment:
A single fail for a workspace stop should not exit the function.

<img width="776" alt="Screenshot 2024-03-09 at 09 24 42" src="https://github.com/daytonaio/daytona/assets/26512078/55f79150-2425-405f-8ca3-8055701f69f9">

In the screenshot above, I had 2 workspaces, one of which failed to start/stop, and then the other did not even get attempted.

Please update this to just log an error if a workspace fails to stop and the loop continues on.

P.S. apply the same change to the start command

Reply:
should i do it for delete too

Reply:
@guptadeepak8 for delete as well, yes. My bad to not including it in the comment.